muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/courses/course.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {connect, Dispatch} from 'react-redux';
+import {i18nType} from '~/reducers/base/i18n';
+import '~/sass/elements/course-description.scss';
+import '~/sass/elements/course.scss';
+import '~/sass/elements/rich-text.scss';
+import '~/sass/elements/application-list.scss';
+import { StatusType } from '~/reducers/base/status';
+import {StateType} from '~/reducers';
+import { ApplicationListItem, ApplicationListItemHeader, ApplicationListItemBody, ApplicationListItemFooter } from '~/components/general/application-list';
+
+interface CourseProps {
+  i18n: i18nType,
+  status: StatusType,
+  course: any
```

Comment:
Please define a type for the data you expect for these courses, if you expect name and description

interface ICourseType { name: string, description: string} and then course: would be ICourseType[]

Make sure there's no similar interface with the same name, just pick a name you think suits.

Reply:
I do not have this